docs/CUSTOMIZATION.md

```diff
@@ -0,0 +1,140 @@
+You can provide a custom color by setting `mcv_selectionColor` in xml, or by calling `setSelectionColor()`.
+
+If you want more control than just color, you can use the [decorator api](DECORATORS.md) to set a [custom selector](CUSTOM_SELECTORS.md).
+
+
+## First Day Of The Week
+
+The default first day of the week is Sunday. You can set a custom day of the week by setting `mcv_firstDayOfWeek` in xml, or by calling `setFirstDayOfWeek()`.
+The xml attribute is an enum of `sunday` through `saturday` and `setFirstDayOfWeek()` accepts values from `java.util.Calendar` such as `Calendar.MONDAY`.
+
+
+## Date Ranges
+
+By default, the calendar displays months for 200 years before today and 200 years after.
+You can specify different minimum and maximum dates by calling `setMinimumDate(CalendarDay)` and `setMaximumDate(CalendarDay)`.
+Passing `null` will reset back to the default 200 years.
+There are also convience methods that accept a `Calendar` or a `Date` object, which convert them to a `CalendarDay` using the relevant `CalendarDay.from()` factory method.
+
+
+## Topbar Options
+
+### Visibility
+
+You can hide or reshow the topbar (arrow buttons and month label) by calling `setTopbarVisible(boolean)`
+
+### Arrow Color
+
+You can change the topbar arrow colors by setting `mcv_arrowColor` in xml, or calling `setArrowColor(int)`.
+This will color the left and right arrow masks using a color filter.
+
+### Arrow Masks
+
+Arrow masks are drawables that will be colored with arrow color and be drawn for previous and next month buttons.
+The color is applied with a color filter using `PorterDuff.Mode.SRC_ATOP`.
+They can be set in xml using `mcv_leftArrowMask` and `mcv_rightArrowMask` or by calling `setLeftArrowMask()` and `setRightArrowMask()`.
+
+
+## Custom Labels
+
+### Header
+
+You can customize the label displayed in the header by setting a custom `TitleFormatter` by calling `setTitleFormatter()`.
+You will be passed a `CalendarDay` with a month and year set, and you can format it to any `CharSequence`.
+The default passes it to a `SimpleDateFormat` with a format of `"MMMM yyyy"`.
+The library provides a `DateFormatTitleFormatter` and `MonthArrayTitleFormatter` as a conveniences.
+
+You can also set a string array resource with `mcv_monthLabels` that will use the `MonthArrayTitleFormatter` to format the title with the months provided.
+
+### WeekDays
+
+You can supply a custom formatter for weekdays with a `WeekDayFormatter` by calling `setWeekDayFormatter()`.
+The default implementation is a `CalendarWeekDayFormatter`, which uses `java.util.Calendar` to get weekday labels.
+We also provide `ArrayWeekDayFormatter`, which uses `CharSequence` array as week day labels.
+You can set `mcv_weekDayLabels` in xml with a string array resource, which will set an `ArrayWeekDayFormatter`.
+
+### DayFormatter
+
+You can set custom day labels by setting a `DayFormatter` by calling `setDayFormatter()`.
```

Comment:
Re-word to "You can set custom day labels by passing a `DayFormatter` to the `setDayFormatter()` method."